designcompose/src/main/java/com/android/designcompose/FrameRender.kt

```diff
@@ -254,13 +272,10 @@ internal fun ContentDrawScope.render(
                 is MeterData.progressBarData -> {
                     val progressBarData = meterData.value
                     if (progressBarData.enabled) {
-                        rectSize =
-                            calculateProgressBarData(
-                                progressBarData,
-                                meterValue,
-                                style.height.pointsAsDp(density).value,
-                                density
-                            )
+                        val progressBarSizeTransform =
```

Comment:
(Style) This would be cleaner to [destructure](https://kotlinlang.org/docs/destructuring-declarations.html) the result, right?

Reply:
Yea good point. I actually remember doing that, but either I lost it or it didn't work because I had incorrect syntax. I don't want to block this PR so I will make a note to fix this in another PR. I have a couple incoming.

Reply:
I remember now -- the variables are already declared so destructuring isn't supported. https://discuss.kotlinlang.org/t/reusing-vars-in-destructuring-declarations/4077